### PR TITLE
Rahti: add certificate to store

### PR DESCRIPTION
**Pull request by vvaltia**
- If certificate exists under /rems/certs/ add it to
   cacerts certificate store before starting rems

# Definition of Done / Review checklist

## Reviewability
- [ ] link to issue
- [ ] note if PR is on top of other PR
- [ ] note if related change in rems-deploy repo
- [ ] consider adding screenshots for ease of review

## API
- [ ] API is documented and shows up in Swagger UI
- [ ] API is backwards compatible or completely new
- [ ] Events are backwards compatible

## Documentation
- [ ] update changelog if necessary
- [ ] add or update docstrings for namespaces and functions
- [ ] components are added to guide page
- [ ] documentation _at least_ for config options (i.e. docs folder)
- [ ] ADR for major architectural decisions or experiments

## Different installations
- [ ] new configuration options added to rems-deploy repository
- [ ] instance specific translations (i.e. LBR kielivara)

## Testing
- [ ] complex logic is unit tested
- [ ] valuable features are integration / browser / acceptance tested automatically

## Accessibility
- [ ] all icons have the aria-label attribute
- [ ] all fields have a label
- [ ] errors are linked to fields with aria-describedby
- [ ] contrast is checked
- [ ] conscious decision about where to move focus after an action

## Follow-up
- [ ] new tasks are created for pending or remaining tasks
- [ ] no critical TODOs left to implement

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -4,6 +4,19 @@ cd rems
 
 [ -z "$COMMANDS" ] && COMMANDS="run"
 
+certfile=$(ls /rems/certs)
```
Comment (opqdonut) [inline docker-entrypoint.sh:7]: should this be documented somewhere?

Comment (vvaltia) [reply docker-entrypoint.sh:7]: Is this sufficient? https://github.com/CSCfi/rems/blob/findata-cert/docs/docker.md

File: docs/docker.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Deploy Rems in docker
```
Comment (opqdonut) [inline docs/docker.md:1]: could mention this file in the Dockerfile

Comment (vvaltia) [reply docs/docker.md:1]: Added a mention to Dockerfile

**Pull request merged**